resolwe_bio/processes/import_data/pathway_map.yml

```diff
@@ -0,0 +1,34 @@
+- slug: upload-metabolic-pathway
+  name: Metabolic Pathway File
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: resolwebio/legacy:latest
+    resources:
+      network: true
+  data_name: Metabolic Pathway
+  version: 0.1.0
+  type: data:pathway
+  category: upload
+  persistence: RAW
+  scheduling_class: interactive
+  description: >
+    Upload pathway json.
+  input:
+    - name: upload
+      label: Pathway file
+      type: basic:file
+      description: >
+        JSON file. Supported extensions: .json.gz
+      validate_regex: '(\.json)(|\.gz|\.bz2|\.tgz|\.tar\.gz|\.tar\.bz2|\.zip|\.rar|\.7z)$'
+  output:
+    - name: pathway
+      label: Pathway json
+      type: basic:json
+  run:
+    runtime: polyglot
+    language: bash
+    program: |
+      re-import {{ upload.file_temp }} {{ upload.file }} "json" "json" 0.5
+      re-save pathway ${NAME}.json
```

Comment:
Add an empty line at the end of the file. You can configure your editor to do that for you automatically.